client/components/dashboard/dashboard-toolbar-directive.js

```diff
@@ -120,14 +128,15 @@ goog.scope(function() {
             return;
           }
 
-          let promise = this.dashboard.deleteDashboard(
-              this.dashboard.current.model);
+          var promise = this.dashboardSvc.deleteDashboard(
+              this.dashboardSvc.current.model);
 
           promise.then(angular.bind(this, function(response) {
             this.openDashboardAdmin();
           }));
 
           promise.then(null, angular.bind(this, function(error) {
+            console.log('failed');
```

Comment:
Either drop the log statement or log a more actionable message (maybe include `error`?).